### PR TITLE
Record the repository slug (owner/repo) to support forks

### PR DESCRIPTION
**Pull request by ghengeveld**
For https://github.com/chromaui/chromatic/issues/3837

Note we must update Chromatic to support the newly added `slug` field on `CreateBuildInput` before publishing the updated CLI.

### CODE REVIEW

File: bin/git/git.js [hunk 1]
```diff
@@ -78,6 +78,12 @@ export async function getVersion() {
   return result.replace('git version ', '');
 }
 
+export async function getSlug() {
+  const result = await execGitCommand(`git config --get remote.origin.url`);
+  const [, slug] = result.match(/([^/:]+\/[^/]+?)(\.git)?/) || [];
```
Comment (tmeasday) [inline bin/git/git.js:83]: Maybe a quick note of what this does? I don't read regex

**Review by tmeasday — APPROVED** [review]
LGTM

File: bin/git/git.js (hunk 1 above)
Comment (ghengeveld) [reply bin/git/git.js:83]: Fair point.

**Review by tmeasday — APPROVED** [review]
Ok by me

**Comment by ghengeveld** [comment]
Released in [v5.6.0](https://www.npmjs.com/package/chromatic).

**Pull request merged**